apps/arweave/src/ar_config.erl

```diff
@@ -721,6 +721,17 @@ parse_options([{<<"shutdown_tcp_connection_timeout">>, Delay} | Rest], Config)
 		parse_options(Rest, NewConfig);
 parse_options([{<<"shutdown_tcp_connection_timeout">>, InvalidValue} | Rest], Config) ->
 	{error, {bad_type, shutdown_tcp_connection_timeout, integer}, InvalidValue};
+parse_options([{<<"shutdown_tcp_mode">>, Mode}|Rest], Config) ->
+	case Mode of
+		"soft" ->
```

Comment:
I think Mode may be a binary string when specificied in the config.json. e.g. `<<"soft">>`